microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -1,24 +1,24 @@
 ---
 description: IE mode and the Microsoft Edge (Chromium) DevTools
-title: Internet Explorer mode and the DevTools
+title: Use the DevTools in Internet Explorer Mode
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 02/12/2021
+ms.date: 09/03/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, ie11, internet explorer 11, ie mode
 ---
-# Internet Explorer mode and the DevTools  
+# Use the DevTools in Internet Explorer Mode  
 
 This article describes how Internet Explorer mode \(IE mode\) integrates with the Microsoft Edge \(Chromium\) DevTools.  
 
 ## Understanding IE mode  
```

Comment:
Link to Enterprise article
What is Internet Explorer (IE) mode?
https://docs.microsoft.com/en-us/deployedge/edge-ie-mode